docs/reference/standard-library.rst

```diff
@@ -38,40 +38,77 @@ Int
 Sequences
 ============
 
-Required for `pluscal procedures <procedure>`.
+Required for `pluscal procedures <procedure>`. A sequence is like a list in any other language. You write it like `<<a, b, c>>`, and the elements can be any other values (including other sequences).
```

Comment:
rSt uses double backticks for code samples, not single backticks.

Reply:
I should add a reference to the section of the docs that has sequences afterwards